### ISSUE TITLE
0.71 release

### ISSUE THREAD

**Issue by reanimus** [ufrisk/puttywincrypt#1]
I saw there was a new release for PuTTY out. Are you planning on making a release here too?

**Comment by reanimus** [comment]
(for the time being I'm using pageant with vanilla 0.71 PuTTY to get the benefits of the new release but I imagine pageant possibly got improvements as well)

**Comment by ufrisk** [comment]
Thanks for the reminder.

Yes, I plan to update to 0.71 once I find the time to look into this - probably this weekend.

**Comment by ufrisk** [comment]
Update,

I've looked into things and there are major changes in core files: `ssh.c`, `sshpubk.c`, and `sshrsa.c` completely changing how keys are handled. This will take quite some more work than a simple patch upgrade.

Unfortunately I'm just so pre-occupied with other things the next 1.5 months so I won't have time to look into this before mid may.

**Comment by reanimus** [comment]
There's another release out now. AFAIK there haven't been breaking changes to pageant so the existing solution of using vanilla putty + wincrypt pageant works still, though.

**Comment by ufrisk** [comment]
Yeah, they haven't changed the protocol between pageant and putty so it still works :) Thats the good news.

They changed pretty much everything else internally though so it would require me quite some time to rework this patch from zero though.

Since my current work don't use wincrypt putty/pageant internally and since it's not very widely used in general I can't justify putting the unpaid time required into rebuilding this patch pretty much from zero when I have too many other things to do already :/

**Comment by svenso** [comment]
We have done some work here: https://github.com/baumerits/putty-pkix
for v0.73.. Even if its optimized for X509 (PKIX) classic rsa-auth using wincrypto apis should also work.
